src/content/en/2020/jamstack.md

```diff
@@ -3,21 +3,265 @@
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
-unedited: true
+featured_quote: Stats suggest almost twice as many Jamstack sites exist now than in 2019. Developers enjoy a better development experience but that doesn't always translate to a better user experience. Let’s work towards continued progress for improving the real-user experience of browsing Jamstack sites.
+featured_stat_1: 154%
+featured_stat_label_1: more mobile websites built with Jamstack in 2020
+featured_stat_2: 1 gram
+featured_stat_label_2: CO2 is emitted by the median Jamstack page load
+featured_stat_3: 38%
+featured_stat_label_3: Jamstack sites are built using Next.js (three times more than any other framework!)
 ---
+
+## Introduction
+
+Jamstack is a relatively new concept of an architecture designed to make the web faster, more secure, and easier to scale. It builds on many of the tools & workflows which developers love and which bring maximum productivity.
+
+The core principles of Jamstack are pre-rendering your site pages and decoupling the frontend from the backend. It relies on the idea of delivering the frontend content hosted separately on a CDN provider that uses APIs (for example, a headless CMS) as its backend if any.
+
+The [HTTP Archive][1] crawls [millions of pages][2] every month and runs them through a private instance of [WebPageTest][3] to store key information on every page. (You can learn more about this in our [methodology][4]). In the context of Jamstack, HTTP Archive provides extensive information on the usage of the frameworks and CDNs for the entire web. This chapter consolidates and analyzes many of these trends.
+
+### Goals: Growth of Jamstack & Real User Perf
+
+The goals of this chapter are to estimate and analyze the growth of the Jamstack sites, the performance of popular Jamstack frameworks, as well as an analysis of real user experience using the Core Web Vitals metadata.
+
+## Adoption: What is the overall level of Jamstack adoption amongst all websites, across desktop and mobile?
+
+Our analysis throughout this work looks at desktop and mobile websites. The vast majority of URLs we looked at are in both datasets, but some URLs are only accessed by desktop or mobile devices. This can cause small divergences in the data, and we thus look at desktop and mobile results separately.
+
+{{ figure_markup(
+  image="jamstack-adoption.png",
+  caption="Jamstack adoption trend.",
+  description="Bar chart showing the increase in the level of Jamstack adoption in 2019 and 2020. Mobile has increased from 0.40% to 0.72%. Desktop has increased from 0.27% to 0.68%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ-zBygubF5NuAXOGScVLhnozB_6Pi_0z4B57cP_I6AfeBuZCGzs3aP_1nPjPp37aNRATRYC6EfOeYX/pubchart?oid=1650360073&format=interactive",
+  sheets_gid="908645975",
+  sql_file="ssg_compared_to_2019.sql"
+  )
+}}
```

Comment:
We need to drop the 3D effect - it's inconsistent with the other chapters. I can take care of this and redoing the images so just an FYI for you.

Reply:
Charts updated in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1632/commits/7f19a62cbac53429817b3b1f649d377633103af2 to:
- Remove 3D effect
- Switch to sentence case
- Use standard height and widths (600 x 371)
- Move CWV labels to outside bar chart as needed when using standard height and widths
- Export hi-res images